tests/integration/components/identity/step1-test.js

```diff
@@ -0,0 +1,81 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, click } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import { Server } from 'ember-cli-mirage';
+import ENV from 'website-my/config/environment';
+import sinon from 'sinon';
+
+module('Integration | Component | identity/step1', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders the identity step1 component', async function (assert) {
+    this.set('setState', (val) => {
+      assert.step(val);
+    });
+
+    await render(hbs`<Identity::Step1 @setState={{this.setState}} />`);
+
+    assert.dom('[data-test-step1-heading]').exists();
+    assert
+      .dom('[data-test-step1-heading]')
+      .hasText('Step 1: Chaincode Generation');
+
+    assert.dom('[data-test-step1-desc]').exists();
+    assert.dom('[data-test-step1-button]').exists();
+  });
+
+  test('it generates the chaincode', async function (assert) {
+    let stub = sinon.stub(window, 'confirm');
+    stub.returns(true);
+
+    let server = new Server({
+      models: {},
+      routes() {
+        this.namespace = ENV.BASE_API_URL;
+
+        this.get('/users/chaincode', { chaincode: 'abcdefghijklmnopqrs' });
+      },
+    });
+
+    this.set('setState', (val) => {
+      assert.step(val);
+    });
+
+    await render(hbs`<Identity::Step1 @setState={{this.setState}} />`);
+
+    assert.dom('[data-test-step1-heading]').exists();
+    assert
+      .dom('[data-test-step1-heading]')
+      .hasText('Step 1: Chaincode Generation');
+
+    assert.dom('[data-test-step1-desc]').exists();
+    assert.dom('[data-test-step1-button]').exists();
+
+    await click('[data-test-step1-button]');
+
+    let resumeTest = this.resumeTest;
+    setTimeout(() => {
+      resumeTest();
+    }, 500);
+    // eslint-disable-next-line ember/no-pause-test
```

Comment:
why this is rule is neglected

Reply:
because that eslint rule does not allow me to write pause test but I want to